src/scripts/clipperUI/clipper.tsx

```diff
@@ -98,7 +99,8 @@ class ClipperClass extends ComponentBase<ClipperState, {}> {
 				isLocalFileAndNotAllowed: true,
 				selectedPageRange: "",
 				shouldAttachPdf: false,
-				shouldDistributePages: false
+				shouldDistributePages: false,
+				shouldShowPopover: false
```

Comment:
This is a tough problem that I think we should punt and track with an issue. I think `shouldShowPopover` isn't an accurate name. The state variable actually reflects if the user clicked on clip while there was an invalid page range.